aspnetcore/fundamentals/choose-aspnet-framework.md

```diff
@@ -1,26 +1,28 @@
 ---
 title: Choose between ASP.NET and ASP.NET Core
 author: rick-anderson
-description: Learn how to choose between ASP.NET and ASP.NET Core.
+description: Explains ASP.NET Core vs. ASP.NET and how to choose between them.
 ms.author: riande
-ms.date: 05/11/2018
+ms.date: 09/11/2018
 uid: fundamentals/choose-between-aspnet-and-aspnetcore
 ---
 # Choose between ASP.NET and ASP.NET Core
 
-No matter the web app you're creating, ASP.NET has a solution for you: from enterprise web apps targeting Windows Server, to small microservices targeting Linux containers, and everything in between.
+ASP.NET Core is a redesign of ASP.NET. This article lists the differences between them.
 
 ## ASP.NET Core
 
 ASP.NET Core is an open-source, cross-platform framework for building modern, cloud-based web apps on Windows, macOS, or Linux.
 
+[!INCLUDE[](~/includes/benefits.md)]
+
 ## ASP.NET
 
-ASP.NET is a mature framework that provides all the services needed to build enterprise-grade, server-based web apps on Windows.
+ASP.NET is a mature framework that provides the services needed to build enterprise-grade, server-based web apps on Windows.
 
 ## Framework selection
 
-Review the table below to determine which framework is most appropriate for your needs.
+The following table compares ASP.NET core to ASP.NET Framework.
```

Comment:
- core --> Core
- ASP.NET Framework --> ASP.NET 4.x

Reply:
@scottaddie 	I prefer ASP.NET 4.x to ASP.NET Framework. I think there's less confusion for customers. Hopefully ASP.NET 4.x will work for the next 3 years when there's new versions of Core and Framework. The PU was adamant it had to be ASP.NET Framework so they might not approve.

cc @tdykstra

Reply:
I agree on preferring ASP.NET 4.x over ASP.NET Framework